lib/elixir/lib/code.ex

```diff
@@ -934,35 +989,76 @@ defmodule Code do
       at compilation time. This can be useful when doing dynamic compilation.
       Defaults to `[]`.
 
-  It returns the new map of compiler options.
+    * `:tracers` - a list of tracers (modules) to be used during compilation. See the
```

Comment:
Should we mark `:no_warn_undefined`, `:tracers`, and `:parser_options` options with `(since v1.10.0)`?

Reply:
Yes!